integration/testrunner/grpc.go

```diff
@@ -119,19 +127,26 @@ func testGRPCEndpoint(ctx context.Context, t *testing.T, g *GRPC) {
 	t.Run("hello_req_unary_success", func(t *testing.T) {
 		t.Parallel()
 		id := uuid.New().String()
+		want := GRPCFields{"/abcxyz.test.Talker/Hello", "gh-access-sa@lumberjack-dev-infra.iam.gserviceaccount.com", "abcxyz.test.Talker"}
 		_, err := g.TalkerClient.Hello(ctx, &talkerpb.HelloRequest{Message: "Some Message", Target: id})
 		if err != nil {
 			t.Errorf("could not greet: %v", err)
 		}
+		time.Sleep(15 * time.Second)
```

Comment:
Relocate this to the "query" func and comment about why this delay is helpful. IIUC, we don't expect logs appear right away so we want to introduce an initial delay to avoid being backed off too much. 

Alternatively, I wonder if getting rid of `retry.NewExponential` and instead using a fixed wait interval is better.

Reply:
It appears that using both fixed wait interval and the retry method is better, as waiting for the log to appear in the bigquery datebase isn't the only possible error.